data_summaries/Indic-Instruct.json

```diff
@@ -0,0 +1,483 @@
+{
+  "indic-instruct-anudesh": {
+    "Unique Dataset Identifier": "indic-instruct-anudesh",
+    "Dataset Name": "indic-instruct-data-v0.1",
+    "Paper Title": "Airavata: Introducing Hindi Instruction-tuned LLM",
+    "Dataset URL": "https://huggingface.co/datasets/ai4bharat/indic-instruct-data-v0.1",
+    "GitHub URL": "https://github.com/AI4Bharat/IndicInstruct",
+    "Hugging Face URL": "https://huggingface.co/datasets/ai4bharat/indic-instruct-data-v0.1/viewer/anudesh",
+    "Papers with Code URL": "",
+    "ArXiv URL": "https://arxiv.org/abs/2401.15006",
+    "Semantic Scholar Corpus ID": "267301288",
+    "Collection": "Indic-Instruct",
+    "Collection URL": "https://github.com/AI4Bharat/IndicInstruct",
+    "Languages": [
+      "English",
+      "Hindi"
+    ],
+    "Task Categories": [
+      "Instructional Question Answering",
+      "Closed-Book Question Answering",
+      "Open-Ended Question Answering",
+      "Natural Language Classification",
+      "Open-Domain Question Answering",
+      "Brainstorming",
+      "Summarization",
+      "Creative Writing",
+      "Instruction Following",
+      "Translation",
+      "Information Extraction"
+    ],
+    "Text Sources": [
+      "crowdsourced"
+    ],
+    "Model Generated": [
+      "IndicTrans2"
```

Comment:
Let's have Robert review if there is any license/model associated with using this model

Reply:
IndicTrans2 is under MIT License, see https://github.com/AI4Bharat/IndicTrans2/blob/main/LICENSE